internal/notification/channels/twilio/channel.go

```diff
@@ -65,3 +72,23 @@ func InitChannel(config Config) channels.NotificationChannel {
 		return nil
 	})
 }
+
+func userAndNotificationIDsFromEvent(event eventstore.Event) (userID, notificationID string) {
+	aggID := event.Aggregate().ID
+
+	// we cannot cast to the actual event type because of circular dependencies
+	// so we just check the type...
+	if event.Aggregate().Type != "notification" {
+		return aggID, ""
```

Comment:
```suggestion
		return "", aggID
```

Reply:
if it's not a notification event, then it's actually the old handler directly processing the user event
i've added a comment